osu.Game/Screens/Play/Player.cs

```diff
@@ -94,6 +94,7 @@ public abstract partial class Player : ScreenWithBeatmapBackground, ISamplePlayb
         public IBindable<bool> LocalUserPlaying => localUserPlaying;
```

Comment:
I'd argue this should no longer be public (or even exist).

Reply:
Yeah, it's gonna touch a lot of tests though (take a quick look and see if you thing it's worth it)

Reply:
As far as I can see the extent of it is mostly asserts, right? If that is accurate then I think it's worth getting rid just to lower the redundancy around here. I'll have to cross-check all of the existing conditional changes anyway to make sure nothing slipped by so that seems like a rather minor inconvenience from both authoring and reviewing angles.